Tsolmon17/Springboard-Coursework
# Amir

### Summary
Cleaned up and merged everything into a single branch that should be a clean slate going forward. I'm not sure if I've used the latest version of some of the exercises so please make sure that everything looks good before we merge into master.

### Bug Fixes/New Features
* Using `-` instead of `_` and removed all spaces in file and directory names. See [this](https://x-equals.com/dashes-versus-underscores/)
* Added a `README.md` file to the root, generally a good idea to have one per directory too
* Added a `.gitignore` file that should stop checking in `.ipynb_checkpoint` folders
* Added a PR template (the one I'm using for this PR)

### How to Verify
Make sure I've used the latest notebooks for all exercises. If not you can checkout this branch, replace the latest file, commit and push.

### Tests
N/A

### Code Reviewer(s)
@Tsolmon17
